src/components/fieldset/template.test.js

```diff
@@ -61,6 +61,15 @@ describe('fieldset', () => {
     expect($legend.text().trim()).toEqual('What is your address?')
   })
 
+  it('allows you to set the initial aria-describedby attribute', () => {
+    const $ = render('fieldset', {
+      describedBy: 'parent-fieldset-hint-id'
```

Comment:
In most cases this _won't_ be the parent fieldset hint ID, so can we make this more generic? Maybe even just…

```suggestion
      describedBy: 'some-id'
```